unity/Assets/Scripts/SimObjPhysics.cs

```diff
@@ -168,7 +176,17 @@ public class PhysicsMaterialValues {
         ContainedObjectReferences.Remove(t);
     }
 
-    public void syncBoundingBoxes(bool forceCacheReset = false) {
+    public void syncBoundingBoxes(
```

Comment:
This name seems a misnomer if I understand correctly? It's not syncing the boxes just setting flags to do so at some point?

Reply:
I don't think so, it will actually regenerate the bounding boxes so long as:
1. The object has moved/rotated or
2. We explicitly pass the `forceCacheReset` argument.

In this PR I added another argument to force the creation of an object-oriented bounding box as sometimes these are not created for certain objects..